### DISCUSSION TITLE
Duplicate files different resolution

### DISCUSSION THREAD

**Discussion by Feejus** [hsoft/dupeguru#1047] [Q&A]
I am starting a discussion as i am not sure even if this is feasible request or it has been done.

Problem: 
As part of everyday life we keep download loads of files including but not limited to videos, for movie files they might be the same files but different resolution. 

The two files could be something like

Myfilename is here to stay.mp4
Myfilename is here to stay 2160.mp4
Myfilename is here to stay 4k.mp4

As you can see above they are identical but the checksum is not and filename is 90% match. 

Q: Is there a way to identify these type of files ? I was thinking that a rule would be if the file is 90% or above match show it as duplicate? Maybe there is a way but i have not seen it or know how to search for those type of  files. 

Thanx

**Comment by Alister-Zeus** [comment]
What I use for video files is video comparer - I't a paid for program but does a good job on video files - the only ones it has issues with are kids type teletubbies/thomas & friends/ cartoons - it'll identify based on video contents so will do wmv/mp4/avi
